dbatools.psd1

```diff
@@ -171,7 +171,8 @@
 		'Resolve-DbaNetworkName',
 		'Test-DbaValidLogin',
 		'Get-DbaMemoryUsage',
-		'Export-DbaAvailabilityGroup'
+		'Export-DbaAvailabilityGroup',
+		'Update-SqlLoginName'	
```

Comment:
Please remove the .psd1 change from the PR. .psd1's can only be made by me since they always create conflicts.